postgrestest.go

```diff
@@ -54,6 +60,7 @@ func New() (*DB, error) {
 	}
 	_, err = db.Exec(`CREATE SCHEMA ` + name)
 	if err != nil {
+		db.Close()
```

Comment:
Wondering if there is a case where CREATE SCHEMA can hang on the data dictionary. That seems like an exceptionally bad day for postgres, but still might be a good idea to:
- log the error returned from Close, just as a diagnostic clue as to what might be going on
- time out in case it hangs (according to https://golang.org/pkg/database/sql/#DB.Close it waits for queries to finish...)